src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  function add(n) {
+    calculator.result += n;
+  }
+
+  function subtract(n) {
+    calculator.result -= n;
+  }
```

Comment:
[CODE KNOWLEDGE] Instead of using the `function` keyword inside the object, you can use the method shorthand syntax for better readability.

BAD EXAMPLE:
 subtract: function(n) {
   this.result -= n;
 },

GOOD EXAMPLE:
 subtract(n) {
   this.result -= n;
 },